### PR TITLE
Core config translations

### PR DESCRIPTION
**Pull request by emlove**
## Description
This PR adds translations for the configuration dashboard and core pages. I figured this was a decent place to start a PR before continuing with the rest of the config pages. The commits are split into smaller chunks if that's easier to review.

### Notes
I don't have a hassbian setup, and it seems like the hassbian config is still pretty WIP, so I haven't done any translations of the dynamic strings, including suite names and suite states.

### CODE REVIEW

File: src/translations/en.json [hunk 1]
```diff
@@ -313,6 +313,78 @@
...
+                "stop": "Stop"
+              }
+            },
+            "push_notifications": {
+              "header": "Configure Push Notifications",
+              "introduction": "Enable this setting to receive push notifications on this device",
+              "push_notifications": "Push Notifications"
+            },
+            "translation": {
+              "header": "Choose a Language",
```
Comment (c727) [inline src/translations/en.json]: language

File: src/translations/en.json [hunk 2]
```diff
@@ -313,6 +313,78 @@
...
+                "heading": "Server Management",
+                "introduction": "Control your Home Assistant server… from Home Assistant.",
+                "restart": "Restart",
+                "stop": "Stop"
+              }
+            },
+            "push_notifications": {
+              "header": "Configure Push Notifications",
+              "introduction": "Enable this setting to receive push notifications on this device",
+              "push_notifications": "Push Notifications"
```
Comment (c727) [inline src/translations/en.json]: notifications

File: src/translations/en.json [hunk 3]
```diff
@@ -313,6 +313,78 @@
...
+              "reloading": {
+                "heading": "Configuration Reloading",
+                "introduction": "Some parts of Home Assistant can reload without requiring a restart. Hitting reload will unload their current configuration and load the new one.",
+                "core": "Reload core",
+                "group": "Reload groups",
+                "automation": "Reload automations",
+                "script": "Reload scripts"
+              },
+              "server_management": {
+                "heading": "Server Management",
```
Comment (c727) [inline src/translations/en.json]: management

File: src/translations/en.json [hunk 4]
```diff
@@ -313,6 +313,78 @@
...
+              "introduction": "Changing your configuration can be a tiresome process. We know. This section will try to make your life a little bit easier.",
+              "validation": {
+                "heading": "Configuration Validation",
+                "introduction": "Validate your configuration if you recently made some changes to your configuration and want to make sure that it is all valid",
+                "check_config": "Check config",
+                "valid": "Configuration valid!",
+                "invalid": "Configuration invalid"
+              },
+              "reloading": {
+                "heading": "Configuration Reloading",
```
Comment (c727) [inline src/translations/en.json]: reloading

File: src/translations/en.json [hunk 5]
```diff
@@ -313,6 +313,78 @@
...
     "panel": {
+      "config": {
+        "header": "Configure Home Assistant",
+        "introduction": "Here it is possible to configure your components and Home Assistant. Not everything is possible to configure from the UI yet, but we're working on it.",
+        "core": {
+          "caption": "General",
+          "description": "Validate your configuration file and control the server",
+          "section": {
+            "core": {
+              "header": "Configuration and Server Control",
```
Comment (c727) [inline src/translations/en.json]: server control

File: src/translations/en.json [hunk 6]
```diff
@@ -313,6 +313,78 @@
...
+              "introduction": "Changing your configuration can be a tiresome process. We know. This section will try to make your life a little bit easier.",
+              "validation": {
+                "heading": "Configuration Validation",
```
Comment (c727) [inline src/translations/en.json]: validation

File: src/translations/en.json [hunk 7]
```diff
@@ -313,6 +313,78 @@
...
+              },
+              "server_management": {
+                "heading": "Server Management",
+                "introduction": "Control your Home Assistant server… from Home Assistant.",
+                "restart": "Restart",
+                "stop": "Stop"
+              }
+            },
+            "push_notifications": {
+              "header": "Configure Push Notifications",
```
Comment (c727) [inline src/translations/en.json]: Configure push notifiations

**Comment by balloob** [comment]
I'll remove the Hassbian control panel. It has never been finished. We took that idea and turned it into Hass.io instead 😎

**Review by balloob — APPROVED** [review]

**Pull request merged**